tests/wicked/advanced/sut/t15_macvtap_legacy.pm

```diff
@@ -0,0 +1,62 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Advanced test cases for wicked
+# Test 15: Create a macvtap interface from legacy ifcfg files
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use base 'wickedbase';
+use strict;
+use warnings;
+use testapi;
+use network_utils 'iface';
+use utils 'zypper_call';
+
+our $macvtap_log = '/tmp/macvtap_results.txt';
+
+sub run {
+    my ($self) = @_;
+    record_info('Info', 'Create a macvtap interface from legacy ifcfg files');
+    my $config     = '/etc/sysconfig/network/ifcfg-macvtap1';
+    my $iface      = iface();
+    my $ip_address = $self->get_ip(type => 'macvtap', netmask => 1);
+    $ip_address =~ s'/'\\/';
```

Comment:
Maybe worth of adding support to `get_ip` for returning escaped `/` via parameter.

Reply:
maybe but for me it looks like a separate PR . this code pattern ( get_ip call and escaping `/` )  used in few other places so I would need to touch to many thing not related to introduction of macvtap test and this is what I am trying to achieve with this PR